src/plugins/index.ts

```diff
@@ -3,15 +3,18 @@ import { BingAdsPluginParams, GoogleAdsPluginParams, PluginParams } from "../sha
 interface ApplyPluginParams extends PluginParams, GoogleAdsPluginParams, BingAdsPluginParams {}
 
 const applyPlugins = (context: ApplyPluginParams) => {
-  switch (context.plugin) {
-    case "googleAds":
-      import("./google").then(({ default: load }) => load(context));
-      break;
-    case "bingAds":
-      import("./bing").then(({ default: load }) => load(context));
-      break;
-    default:
-      console.log("Plugin not found");
+  const plugins = context.plugin.split(",");
+  for (const plugin of plugins) {
+    switch (plugin) {
+      case "googleAds":
+        import("./google").then(({ default: load }) => load(context));
+        break;
+      case "bingAds":
+        import("./bing").then(({ default: load }) => load(context));
+        break;
+      default:
+        console.log("Plugin not found");
+    }
   }
```

Comment:
I really like the simple implementation. Looks Good!

Tested the Google Ads functionality: Successful
![image](https://user-images.githubusercontent.com/69985528/212379250-2a5d9a4d-69fb-474a-812e-1621f072e9f2.png)

Tested the Bing Ads functionality: Successful
![image](https://user-images.githubusercontent.com/69985528/212379329-627ff86a-c136-4e42-a3ae-4a9f0b0d5094.png)

I made an additional commit to update the Readme regarding this contribution. Thanks so much!